omim2obo/utils/utils.py

```diff
@@ -2,8 +2,9 @@
 from typing import Dict, List, Optional, Union
 
 import pandas as pd
+from rdflib import URIRef
 
-from omim2obo.config import DISEASE_GENE_EXCLUSIONS_PATH
+from omim2obo.config import DISEASE_GENE_EXCLUSIONS_PATH, DISEASE_GENE_PROTECTIONS_PATH
```

Comment:
`DISEASE_GENE_PROTECTIONS_PATH` --> `DISEASE_GENE_PROTECTIONED_PATH`

Reply:
Renamed and pushed along w/ the other changes.